baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/ValidateConstantMessageTests.java

```diff
@@ -109,8 +109,11 @@ public void negative() throws Exception {
                         "  void f(boolean bArg, int iArg, Object oArg, Integer[] arrayArg, "
                             + "Collection<String> collectionArg, Map<String, String> mapArg, String stringArg, "
                             + "Iterable<String> iterableArg, double dArg) {",
+                        "    String localConstant = \"constant\";",
                         "    Validate.isTrue(bArg, \"message %d\", 123L);",
                         "    Validate.isTrue(bArg, \"message %f\", 0.0);",
+                        "    Validate.notBlank(stringArg, \"message %s\", compileTimeConstant);",
+                        "    Validate.notBlank(stringArg, \"message %s\", localConstant);",
```

Comment:
haha yes. I switched to just using `stringArg` rather than defining a new local constant